stylesheets/decode/types/_string.scss

```diff
@@ -21,8 +21,7 @@
     // If string is not empty
     $string: str-slice($temp, 1, $end - 1);
 
-    $cr: "
-    ";
+    $cr: "";
```

Comment:
Fair enough, that's a shame. For what it's worth, it seems to be working perfectly for me :) But frankly I have no idea what this code is supposed to be doing.